content/chainguard/chainguard-images/debugging-distroless-images.md

```diff
@@ -0,0 +1,143 @@
+| `php`                	| X  	| X      	| X      	|
+| `apk-tools`          	|    	| X      	|        	|
+| `bash`               	|    	| X      	|        	|
+| `busybox`            	|    	| X      	|        	|
+| `git`                	|    	| X      	|        	|
+| `composer`           	|    	| X      	|        	|
+| `php-fpm`            	|    	|        	| X      	|
+
+You can find similar comparison tables for all Chainguard Images on their respective "Image Variants" page, on the [Images Reference]() section of the documentation.
+
+Having a package manager and the ability to log into the image to debug any issues is very important at development time, but becomes unnecessary (and less safe) when talking about production environments. That's why we recommend using a distroless variant for production workloads.
+
+### Why you Should Avoid Using Dev Images on Production
+Although the `-dev` image variants have similar security features as their distroless versions, such as complete SBOMs and signatures, they are not designed to run on production. Because they feature additional software such as package managers and debugging tools, they have a bigger attack surface, with tools that facilitate tampering with the container. The general recommendation is to use the `-dev` variants only to build the application and then copy all application artifacts into a distroless image, which will result in a final container image that has a minimal attack surface and won't allow package installations or logins.
+
+That being said, it's worth noting that the `-dev` variants of Chainguard Images are still **more secure** than many popular container images based on fully-featured operating systems such as Debian and Ubuntu, because they carry less software, follow a more frequent patch cadence, and offer attestations for what is included.
+
+### Language Ecosystem Guides
+The following guides show how to use these `-dev` images in combination with their distroless variants in order to build a final image that is also distroless, but contains everything the application needs to run:
+
+- [Getting Started with the Python Chainguard Image](/chainguard/chainguard-images/reference/python/getting-started-python/)
+- [Getting Started with the Ruby Chainguard Image](/chainguard/chainguard-images/reference/ruby/getting-started-ruby/)
+- [Getting Started with the Go Chainguard Image](/chainguard/chainguard-images/reference/go/getting-started-go/)
+- [Getting Started with the Node Chainguard Image](/chainguard/chainguard-images/reference/node/getting-started-node/)
+- [Getting Started with the PHP Chainguard Image](/chainguard/chainguard-images/reference/php/getting-started-php/)
+
+Check also the guide on [Creating Wolfi Images with Dockerfiles](/open-source/wolfi/wolfi-with-dockerfiles/) for guidance on how to build a custom image that can be used for development and debugging.
+
+
+## 2. Using Ephemeral (or Debug) Containers
+
```

Comment:
I would add a transition here, something like "Another method for debugging is to use an ephemeral container to achieve ..."